app/services/idv/financials_step.rb

```diff
@@ -29,6 +29,10 @@ def vendor_errors
       vendor_validator.errors if form_valid?
     end
 
+    def vendor_reasons
+      vendor_validator.reasons if form_valid?
```

Comment:
If the form itself is not valid (based on our internal validations), then we don't want to contact the vendor at all because it would be a waste of resources. The Form validation errors are already captured at this point. If the Form is valid, then we perform the second validation with the vendor, and we capture the errors.

However, I question the implementation of `reasons` in the proofer gem. It seems to me that the reasons should be part of the errors, just like they are in Rails. I would expect to see something like this:
```ruby
result = {
  success: false,
  idv_attempts_exceeded: false,
  errors: {
    first_name: ['The name was suspicious.']
  }
}
```
as opposed to this:
```ruby
result = {
  success: false,
  idv_attempts_exceeded: false,
  errors: {
    first_name: ['Unverified first name.']
  },
  reasons: ['The name was suspicious']
}
```
where it's not clear which fields the reasons are referring to.